packages/app/src/components/nomination/nominationShow.js

```diff
@@ -1,48 +1,46 @@
-import React, { useEffect, useState } from 'react';
-import NominationBanner from './nominationBanner'
-import nominationsAPI from '../../utils/API/nominationsAPI';
-import NominationInfo from '../nominationInfo';
-import './style.css';
+import React, { useContext, useEffect, useState } from 'react';
+import NominationBanner from './nominationBanner';
+import { ActiveNominationContext } from '../../utils/context/ActiveNominationContext';
+import { NominationsDataContext } from '../../utils/context/NominationsContext';
 
+import './style.css';
 
-const NominationShow = ({ match: { params: { id } } }) => {
-  const [NominationData, setNominationData] = useState({})
-  const [errorMessage, setErrorMessage] = useState(null)
+const NominationShow = ({
+  match: {
+    params: { id },
+  },
+}) => {
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [activeNomination, setActiveNomination] = useContext(
+    ActiveNominationContext
+  );
+  const [NominationsData, setNominationsData] = useContext(
+    NominationsDataContext
+  );
 
   useEffect(() => {
-    nominationsAPI.fetchNomination(id)
-      .then(function (response) {
-        const nomination = response.data.nomination
-        setNominationData(nomination)
-      })
-      .catch(function (err) {
-        setErrorMessage(err.response)
-      })
+    if (NominationsData) {
+      NominationsData.filter((nomination) => {
+        if (nomination.id === id) {
+          return setActiveNomination(nomination);
+        }
+        if (!id) {
+          setErrorMessage(true);
```

Comment:
What's the reason for just setting the error message to `true` here?

Reply:
If it somehow goes to the nomination page without history in the url or setting the url prop it will still render the page but with an error